input/data/2005/2005A%26A...442....1A/tev-000060.yaml

```diff
@@ -1,4 +1,21 @@
 source_id: 60
 reference_id: 2005A&A...442....1A
 
-# TODO: add data
+data:
+  livetime: 48.6
+  significance: 13.8
+
+pos:
+  ra: {val: 13h2m49.3s, err: 2.3s, err_sys: }
+  dec: {val: -63d49m53s, err: 17s, err_sys: }
+
+morph:
+  type: gauss
+
+spec:
```

Comment:
In the abstract and on page 4, right side in the middle.
Does pl need a erange or is it optional?